### DISCUSSION TITLE
Cursor freezes consistently

### DISCUSSION THREAD

**Discussion by Boo-BooLazer** [ppy/osu#19655] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Having an issue where my cursor will freeze consistently on the latest update for lazer. This never happened before, and the game ran just fine. I am using no special drivers and the issue only applies to my pen tablet. I have a Wacom CTH-460 and Windows correctly registers inputs, it is only lazer that I am having problems with. I have a clip attached below of the problem. Once I switch to the tablet it starts freezing. Halfway through the clip I turn off the "High Precision Mouse" setting and it solves the problem. You may notice that I have the tablet setting enabled in the video, but I have already tried messing with that to no avail. Sorry for the horrible quality, github has a upload limit. Thanks for reading this.

### Screenshots or videos

https://user-images.githubusercontent.com/94070063/183558885-1bd63e65-6c00-47d9-bc79-f4bdfe3c3312.mp4

### Version

2022.731.1

### Logs

[runtime.log](https://github.com/ppy/osu/files/9286561/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9286562/updater.log)
[database.log](https://github.com/ppy/osu/files/9286564/database.log)
[network.log](https://github.com/ppy/osu/files/9286565/network.log)
[performance.log](https://github.com/ppy/osu/files/9286566/performance.log)

**Comment by peppy** [comment, marked as answer]
Something on your PC is blocking the windows input loop.

Please try closing all other applications.

**Reply by Boo-BooLazer** [reply]
Just did that, still no luck.

**Reply by peppy** [reply]
Tracking at https://github.com/ppy/osu/issues/19960. Please disable "high precision cursor" for now.
